transitions/registration.js

```diff
@@ -272,5 +252,43 @@ module.exports = {
                 callback();
             }
         });
+    },
+    addPatient: function(options, callback) {
+        var doc = options.doc,
+            db = options.db,
+            audit = options.audit,
+            id = doc.patient_id,
+            patientNameField = (options.params && options.params.length && options.params[0]) || 'patient_name';
+
+        utils.getRegistrations({
+            db: db,
+            id: id
+        }, function(err, registrations) {
+            if (err) {
+                return callback(err);
+            }
+            if (registrations.length) {
+                // patient already registered, no action required
+                return callback();
+            }
+            db.medic.view('medic-client', 'people_by_phone', {
+                key: [ doc.from ],
+                include_docs: true
+            }, function(err, result) {
+                if (err) {
+                    return callback(err);
+                }
+                var contact = _.result(_.first(result.rows), 'doc');
+                // create a new patient with this patient_id
+                var patient = {
+                    name: doc.fields[patientNameField],
+                    parent: contact && contact.parent,
```

Comment:
Actually no the reminders won't be lost. The reminders, validations, automatic responses, etc are based on the report _not_ the patient/contact/parent. So the app will continue to function perfectly.

I thought for a while about not creating the patient at all, but doing it here still gives admins the chance to see the patient information grouped together and makes the problem visible. It also makes it possible for managers to assign the patient to a CHP.

Furthermore I expect most instances will be configured to reject submissions from unknown sources so those submissions will never get here.